### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Primdahl26**
This readme is getting kinda old..

Updated the most recently changed parts

### CODE REVIEW

**Review by Pankai222 — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -30,43 +30,45 @@ Authors 👷:
 Here's how to get the application up and running for development
 
 ### Prerequisites
-* Got Git installed
-* Got Docker installed
+* Got `Git` installed
+* Got `Docker` installed
```
Comment (AndreasPB) [inline README.md:34]: I think we should stick to using backticks for commands/code snippets and maybe just use bold if we want to highlight names/words
```suggestion
* Got **Git** installed
* Got **Docker** installed
```

**Review by AndreasPB — APPROVED** [review]
LGTM with 1 change

File: README.md (hunk 1 above)
Comment (Primdahl26) [reply README.md]: Yeah i changed it.

**Pull request merged**